experimenter/experimenter/experiments/models.py

```diff
@@ -1347,6 +1353,124 @@ def get_branch_data(self, analysis_basis, selected_segment):
 
         return branch_data
 
+    def formattedAnalysisPointComparator(self, point):
+        wi = point.get("window_index")
+        return int(wi) if wi is not None else 0
+
+    def get_metric_area_data(
+        self, metrics, analysis_basis, segment, reference_branch, window="overall"
+    ):
+        metric_data = {}
+        metric_area_data = {"metrics": metrics, "data": metric_data}
+
+        branches = self.sorted_branches
```

Comment:
I'm guessing `sorted_branches` here is a property method so storing it like this is to prevent having to rerun the computation multiple times?